src/UserService/appsettings.json

```diff
@@ -14,6 +14,9 @@
     "Id": "CBE144F5-AD53-4D0D-B6A1-39133E3F7D2D",
     "Name": "UserService"
   },
+  "EmailEngineConfig": {
+    "ResendInterval": 60
```

Comment:
It is better to use postfix "InMinutes" or "InSeconds", because now it is unclear